app/launch.py

```diff
@@ -321,9 +321,10 @@ def launch(username, password=None, password_hash=None, imagetype=None,
         # hash the linux password and store as environment variable
         # used to connect local IDEs (e.g., atom, vscode) to remote jupyter sessions
         h = hashlib.new('sha256')
+        h.update(password.encode())
```

Comment:
The flask app does not use the `password` parameter anymore. It uses the `password_hash` parameter. The flask app will break if you try using the `password` param here. It will throw an `AttributeError` since `password` will be None and NoneTypes do not have an attribute `encode`. Keep this down below.

Reply:
Ah shoot, good catch. Unfortunately though, using `launch.py` from the command line doesn't work the way the code is currently written.